src/ray/raylet/scheduling_queue.h

```diff
@@ -116,21 +116,35 @@ class SchedulingQueue {
   void MoveTasks(std::unordered_set<TaskID> tasks, TaskState src_state,
                  TaskState dst_state);
 
+  class TaskQueue {
+  public:
+    TaskQueue() {}
```

Comment:
need to have a destructor, which flushes all elements from the task list and the task hashmap. Otherwise we might have a memory leak here.

Reply:
thanks, revised.